doc/src/examples/arithmetic/cross_product

```diff
@@ -0,0 +1,47 @@
+
+// Boost.Geometry
+// QuickBook Example
+
+// Copyright (c) 2020, Aditya Mohan
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+// Calculate the cross product of two points
+
+
+
+#include <iostream>
+#include <boost/geometry.hpp>
+#include <boost/geometry/arithmetic/cross_product.hpp>
+
+namespace bg = boost::geometry; /*< Convenient namespace alias >*/
+
+int main()
+{
+     //Example 1
+     
+     bg::model::point<double, 2, bg::cs::cartesian> p1(7.0, 2.0);
+     bg::model::point<double, 2, bg::cs::cartesian> p2(4.0, 5.0);
+           
+     bg::model::point<double, 2, bg::cs::cartesian> r1;
+       
+     r1 = bg::cross_product(p1,p2);
+
+     std::cout << "Cross Product 1: "<< r1.get<0>() << std::endl;
```

Comment:
in this case cross product is just the determinant. I do not know if this is the best case for example since it is a bit singular but in any case please add a comment about second coordinate of r1 that is undefined